packages/manager/modules/web-paas/src/components/modify-plan/modify-plan.utils.js

```diff
@@ -0,0 +1,19 @@
+import { buildURL } from '@ovh-ux/ufrontend/url-builder';
+
+export const addProjectResolves = {
+  plans: /* @ngInject */ (catalog) => catalog.plans,
+  deleteUser: /* @ngInject */ ($state) => (customer) =>
+    $state.go('web-paas.add.delete-user', {
+      customer,
+    }),
+  goBack: /* @ngInject */ (goToWebPaas) => goToWebPaas,
+  getOrdersURL: /* @ngInject */ () => (orderId) =>
+    buildURL('dedicated', '#/billing/orders', {
```

Comment:
I think we now need to use `coreConfig` instead since https://github.com/ovh/manager/pull/4814

Reply:
Solved here #4920